### DISCUSSION TITLE
Allow more Management Plugin log configurations

### DISCUSSION THREAD

**Discussion by inikulshin** [rabbitmq/rabbitmq-server#7316] [Ideas]
Currently, only `http_log_dir` is supported.
However, most (all?) `lager` file backend settings (name, date, size, count, formatter, ...) are relevant in this case. Especially `count` to avoid creating thousands of log files (that's the reason I'm starting this discussion).

I suppose, the best solution will be to use `lager` in `webmachine_log.erl` instead of `file:write`.

**Comment by michaelklishin** [comment]
Not `lager` but the standard Erlang 25+ logger, please. It supports formatting configuration to some extent. Not everything in the standard logger works exactly like in Lager but in general, they are close to feature parity from what I know.
